source/c_threedigrid.rst

```diff
@@ -0,0 +1,59 @@
+.. _threedigrid:
+
+Threedigrid
+===========
+
+The Python package for low level access to the 3DI core administration file and result file.
```

Comment:
Iets meer info voor niet Python gebruikers, wat is het en wat kun je ermee? En wanneer heb je het niet nodig?